php_apcu/manifest.json

```
@@ -0,0 +1,37 @@
+{
+  "display_name": "php_apcu",
+  "maintainer": "noname@withgod.jp",
+  "manifest_version": "1.0.0",
+  "name": "php_apcu",
+  "metric_prefix": "php_apcu.",
+  "metric_to_check": "",
```

```suggestion
  "metric_to_check": "php_apcu.cache.mem_size",
```

Including a metric to check will auto-install this tile when metrics are detected

---

fixed c4935f725130efef0b2ec0ec5b3c0a44a7ac860d